molgenis-data/src/main/java/org/molgenis/data/meta/MetaDataServiceImpl.java

```diff
@@ -272,8 +272,10 @@ public Package getPackage(String string)
 			throw new UnknownEntityException(format("Unknown entity [%s]", entityMeta.getName()));
 		}
 
-		// add/update attributes, attributes are deleted when deleting entity meta data if no more references exist
-		Iterable<AttributeMetaData> ownAttrs = entityMeta.getOwnAttributes();
+		// FIXME Is this true??
```

Comment:
Remove this comment from code and add please add this comment into the TODO list MOLGENIS 2.0.